services/server/src/fawry/lib/signature.ts

```diff
@@ -9,12 +10,21 @@ import {
 } from "@/fawry/types/requests";
 
 function generateSignature(...data: unknown[]) {
+  // concatenate data values
```

Comment:
This file is missing the payment using card token: https://developer.fawrystaging.com/docs/card-tokens/create-use-token#create-card-token-head (which we will use)